Buffs/Taric/Radiance/Radiance.cs

```diff
@@ -0,0 +1,35 @@
+﻿using GameServerCore.Enums;
+using GameServerCore.Domain;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using LeagueSandbox.GameServer.GameObjects.Stats;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Radiance
+{
+    internal class Radiance : IBuffGameScript
+    {
+        private StatsModifier _statMod;
+        private IBuff _healBuff;
+
+        public void OnActivate(IObjAiBase unit, ISpell ownerSpell)
+        {
+            _statMod = new StatsModifier();
+            _statMod.AttackDamage.FlatBonus += 10f + ownerSpell.Level * 20;
+            _statMod.AbilityPower.FlatBonus += 10f + ownerSpell.Level * 20;
+            unit.AddStatModifier(_statMod);
+            var a =_healBuff = AddBuffHudVisual("Radiance", 10.0f, 1, BuffType.COMBAT_ENCHANCER, unit,10.0f);

```

Comment:
These 2 lines seems to me very wrong. Why do you override _healBuff with 2 buffs to vars that never used?

Reply:
that's my fault, i will remove "vars" , and another thing,
if I use 2 AddBuffHudVisual(); in `_healBuff` , will it have problems?